deployment/stacks/parent.ts

```diff
@@ -61,49 +45,27 @@ export class ParentStack extends Stack {
 
     const { tables } = createDataComponents(commonProps);
 
-    if (isLocalStack) {
-      createApiComponents({
-        ...commonProps,
-        tables,
-        vpc,
-        kafkaAuthorizedSubnets,
-      });
-      /*
-       * For local dev, the LocalStack container will host the database and API.
```

Comment:
Ben wanted to add this note for SEDS.  I think we should keep it.  Thoughts?

Reply:
Fixed.